src/coreComponents/common/format/table/TableFormatter.cpp

```diff
@@ -475,9 +475,8 @@ void TableTextFormatter::stretchColumnsByMergedCellsWidth( stdVector< size_t > &
       {
         CellLayoutRow & row = tableGrid[rowId];
         TableLayout::CellLayout const & cell = row.cells[columnId];
-        TableLayout::CellLayout const & leftCell = row.cells[columnId - 1];
         bool const isToMergeWithRight = cell.m_cellType == CellType::MergeNext;
-        bool const isToMergeWithLeft = columnId > 0 && leftCell.m_cellType == CellType::MergeNext;
+        bool const isToMergeWithLeft = columnId > 0 && row.cells[columnId - 1].m_cellType == CellType::MergeNext;
```

Comment:
your solution is fine, but this would be better imo....just because we aren't relying on the early exit after the lhs of `&&` fails. 
```suggestion
bool const isToMergeWithLeft = ( columnId > 0 ) ? ( row.cells[columnId - 1].m_cellType == CellType::MergeNext ) : false; 
```

Reply:
thanks @rrsettgast I agree with this